cmd/juno-cli/cli/root.go

```diff
@@ -121,12 +120,16 @@ func initConfig() error {
 		}
 		viper.SetConfigFile(filepath.Join(config.Dir, "juno.yaml"))
 		err = viper.ReadInConfig()
-		errpkg.CheckFatal(err, "Failed to read in Config after generation.")
+		if err != nil {
+			Logger.With("Error", err).Error("Failed to read config file")
+		}
```

Comment:
This is not equivalent to the behaviour that existed before. Whenever the error package calls check fatal the binary would stop running. However, using `Logger.Error` will not stop the execution of the binary. Therefore you should use `Logger.Fatal/f()` instead.